README.md

```diff
@@ -173,6 +173,36 @@ console.log(await promptWithCalculator("What is 2 + 2?"));
 
 We'll likely explore more specific APIs for tool- and function-calling in the future; follow along in [issue #7](https://github.com/webmachinelearning/prompt-api/issues/7).
 
+### Structured output or JSON output
+
+To help with programmatic processing of language model responses, the prompt API supports structured outputs defined by a JSON schema.
+
+```js
+const session = await ai.languageModel.create();
+
+const responseJSONSchemaObj = new AILanguageModelResponseSchema({
+  "type" : "object",
+  "required" : ["Rating"],
+  "additionalProperties" : false,
+  "properties" : {
+    "Rating": {
+      "type": "number"
+      "minimum": 0,
+      "maximum": 5
+    }
+  }
+});
+
+// Prompt the model and wait for the json response to come back.
+const result = await session.prompt("Summarize this feedback into a rating between 0-5: "+
+  "The food was delicious, service was excellent, will recommend.",
+  {responseJSONSchema : responseJSONSchemaObj}
+);
+console.log(result);
```

Comment:
Will `result` be a JS value, or a string that needs to be `JSON.parse()`ed?

A JS value would be nicer for developers, but I can't see how to make it work with `promptStreaming()`, so maybe it should be a string?

Reply:
Yup would preclude streaming, and we would have two different types of return objects from the same API. Leaving it as string.

Reply:
Can you explain this in the explainer?

Reply:
done